events/sick_man_events.txt

```diff
@@ -0,0 +1,664 @@
+﻿namespace = sick_man
+# start event
+sick_man.1 = {
+	type = country_event
+	placement = root
+	title = sick_man.1.t
+	desc = sick_man.1.d
+	flavor = sick_man.1.f
+	duration = 3
+	event_image = {
+		video = "unspecific_factory_closed"
+	}
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	icon = "gfx/interface/icons/event_icons/event_skull.dds"
+	trigger = {
+	}
+	immediate = {
+		set_variable = {
+			name = sick_man_var
+			value = 0
+		}
+		set_variable = {
+			name = sick_man_separatist_var
+			value = 0
+		}
+		add_journal_entry = {
+			type = je_sick_man_main
+		}
+	}
+	option = {
```

Comment:
not default?

Reply:
Doesn't seem like it needs it since it's only the one entry?

Reply:
one option*